andela/travel_tool_front
# #161574495 Fix notification count issue

#### What does this PR do?
This PR fixes the issue of notification count not reducing when the `notification read icon` of a notification is clicked.

#### Description of Task to be completed?
- Create a `method` to handle marking single notification as read.

#### How should this be manually tested?
- Clone the repo - `git clone https://github.com/andela/travel_tool_front.git`
- Enter into the project directory - `cd travel_tool_front`
- Install dependencies and start the  app - `make start`
-  [Start](https://github.com/andela/travel_tool_back/pull/139)  the backend.
- Navigate to [travela](http://travela-local.andela.com:3000) in your browser and login into the app. 
- Make a new request
- Open the notification panel
- Click the 'envelope' icon to mark the notification as `read`
- The `notification count` should decrease by one

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#161574495](https://www.pivotaltracker.com/story/show/161574495)

#### Screenshots (if appropriate)
![notification](https://user-images.githubusercontent.com/27569442/47789967-503d1700-dd16-11e8-920c-e5b952c70d69.gif)

#### Questions:
N/A
